src/content/smart-contracts/index.md

```diff
@@ -29,47 +23,43 @@ Alice and Bob are having a bicycle race. Let's say Alice bets Bob $10 that she w
 
 - You select a product
-- The vending machine returns the amount required to purchase the product
-- You insert the correct amount
-- The vending machine verifies you have inserted the correct amount
-- The vending machine dispenses the product of choice
+- The vending machine displays the price
+- You pay the price
+- The vending machine verifies that you paid the right amount
+- The vending machine gives you your item.
 
 The vending machine will only dispense your desired product after all requirements are met. If you don't select a product or insert enough money, the vending machine won't give out your product.
 
 ### Automatic execution {#automation}
 
-One of the most significant benefits smart contracts have over regular contracts is that the outcome is automatically executed when the contract conditions are realized. There is no need to wait for a human to execute the result. In other words: smart contracts remove the need for trust.
+The main virtue of a smart contract is that it automatically executes the outcome when its conditions are met. There is no need to wait for a human to execute the result. This removes the need for a  trusted intermediary.
 
-For example, you could write a smart contract that holds funds in escrow for a child, allowing them to withdraw funds after a specific date. If they try to withdraw the funds before the specified date, the smart contract won't execute. Or, you could write a contract that automatically gives you a digital version of a car's title when you pay the dealer.
+For example, you could write a smart contract that holds funds in escrow for a child, allowing them to withdraw funds after a specific date. If they try to withdraw before that, the smart contract won't execute. Or you could write a contract that automatically gives you a digital version of a car's title when you pay the dealer.
 
 ### Predictable outcomes {#predictability}
 
-The human factor is one of the biggest points of failure with traditional contracts. For example, two individual judges may interpret a traditional contract in different ways. Their interpretations could lead to different decisions getting made and disparate outcomes. Smart contracts remove the possibility of different interpretations. Instead, smart contracts execute precisely based on the conditions written within the contract's code. This precision means that given the same circumstances, the smart contract will produce the same result.
+Traditional contracts are risky because they rely on people to implement them. For example, two judges might interpret a contract differently, which could lead to inconsistent decisions and unequal outcomes. Smart contracts remove this possibility. Instead, smart contracts execute precisely based on the conditions written within the contract's code. This precision means that given the same circumstances, the smart contract will produce the same result.
```

Comment:
Wow, the original was pretty clunky.

```suggestion
Traditional contracts are risky because they rely on people to implement them. For example, two judges might interpret a contract differently, leading to inconsistent decisions and unequal outcomes. Smart contracts remove this possibility. Instead, smart contracts execute precisely based on the conditions written within the contract's code. This precision means that given the same circumstances, the smart contract will produce the same result.
```

Reply:
I feel like we can do better than 'risky' here

Reply:
I agree -- maybe
> Traditional contracts rely on people to implement them, which introduces a potential point of failure.

Or something like that

Reply:
That works. 

I'm kinda thinking using the word 'humans' over 'people' might be good here but I also think it might be a little bit too detached. Any thoughts?

Reply:
'humans' is good! My company (https://www.globalincomecoin.com/) uses the word human a lot instead of 'users' -- I think because users might have bad connotations? -- so that seems pretty natural to me

Reply:
Good with this 👍 Similarly, I think use of "human" here is appropriate

Reply:
```suggestion
Traditional contracts are ambiguous because they rely on humans to interpret and implement them. For example, two judges might interpret a contract differently, which could lead to inconsistent decisions and unequal outcomes. Smart contracts remove this possibility. Instead, smart contracts execute precisely based on the conditions written within the contract's code. This precision means that given the same circumstances, the smart contract will produce the same result.
```